src/resources/your-label/section-one/view.html

```diff
@@ -503,6 +506,11 @@ <h2>Exemptions</h2>
       </a>
     </li>
   </ul>
+</section>
+
+<section>
+  <a name="extra-requirements"></a>
```

Comment:
Why an empty tag?

Reply:
The other sections have the same, for example `<a name="exemptions"></a>`, which I think is for the links at the top of the page